src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -205,10 +205,38 @@ public static bool MonitoringIsEnabled
         public event ResolveEventHandler ResourceResolve { add { } remove { } }
         //public void SetPrincipalPolicy(System.Security.Principal.PrincipalPolicy policy) { }
         /*
-        public void SetThreadPrincipal(System.Security.Principal.IPrincipal principal) 
-        { 
+        public void SetThreadPrincipal(System.Security.Principal.IPrincipal principal)
+        {
             if (principal == null) throw new ArgumentNullException("principal");
-            throw new PlatformNotSupportedException(); 
+            throw new PlatformNotSupportedException();
         }*/
     }
-}
\ No newline at end of file
+    public abstract class ContextBoundObject : System.MarshalByRefObject
+    {
+        protected ContextBoundObject() { }
+    }
+    public partial class ContextMarshalException : SystemException
+    {
+        public ContextMarshalException()
+        {
+            HResult =  HResults.COR_E_CONTEXTMARSHAL;
```

Comment:
Nit: there's an extra space after the equal sign... same in the recurrences below